bodhi/server/models/models.py

```diff
@@ -854,6 +854,7 @@ def edit(cls, request, data):
             comment += '\n\nRemoved build(s):\n'
             for removed_build in removed_builds:
                 comment += "\n- %s" % removed_build
+            comment += '\n\n\nAll karma has been reset.'
```

Comment:
Doesn't karma get reset on new builds too? Also, I'd recommend just saying "Karma has been reset." instead of using the word all, just to be less verbose.